poc/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        classpath("com.autonomousapps:dependency-analysis-gradle-plugin:1.14.1-SNAPSHOT")
+    }
+}
+
+apply(plugin = "com.autonomousapps.dependency-analysis")
```

Comment:
Easier to access POC if you're curious. I'll make sure this is not in my branch's history before merge that way it doesn't clutter anything.